src/index.ts

```diff
@@ -41,18 +41,21 @@ const getTrayIcon = () => {
 const windowDimsSettingsKey = 'overlayed.window'
 
 const allocMainWindow = () => {
+  
+  // create our window
+  mainWindow = new CompositeWindow()
```

Comment:
__tldr: i'd prefer to put this back to where we pass dimensions directly to the window__

we were actually passing windowDims here for a reason - if we give the correct dimensions (and position) on creation, the window is created at that location. If we refactor to code like this, we'll actually get a flicker where the window is created at the incorrect position (electron default) and then moved to this position with the `setSize` and `setPosition` code below.

Reply:
Ok, i'll fix these next monday 😄